src/main/java/org/sciborgs1155/robot/flywheel/Flywheel.java

```diff
@@ -0,0 +1,88 @@
+package org.sciborgs1155.robot.flywheel;
+
+import static edu.wpi.first.units.Units.Volts;
+import static org.sciborgs1155.robot.flywheel.FlywheelConstants.*;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine.Direction;
+import java.util.function.DoubleSupplier;
+import monologue.Logged;
+import monologue.Annotations.Log;
+
+import org.sciborgs1155.robot.Robot;
+
+public class Flywheel extends SubsystemBase implements AutoCloseable, Logged {
```

Comment:
`Flywheel` is actually a bad name for this, just call it `Shooter`, same for the io classes, constant file, etc